src/services/MenuStore.ts

```diff
@@ -276,4 +276,12 @@ export class MenuStore {
     this._unsubscribe();
     this._hashUnsubscribe();
   }
+
+  private static escapeQuotes(str: string) : string {
+    if (typeof str != 'undefined') {
+      str = str.replace(/["\\]/g, '\\$&');
+    }
+
+    return str;
+  }
```

Comment:
```suggestion
  private static escapeQuotes(str: string) : string {
    return str.replace(/["\\]/g, '\\$&');
  }
```
Also `src/utils/helpers.ts` much better place for this function, near existing `unescapeHTMLChars`